### ISSUE TITLE
Set browser window to full-screen mode on initial load

### ISSUE THREAD

**Issue by McPringle** [McPringle/apus#36]
When the browser loads the social wall, it should automatically switch to full-screen mode (like when starting a presentation with Google Presentations).

**Comment by McPringle** [comment]
It should only switch to full-screen mode when a parameter was added to the URL. Example:
https://really-fancy-url/?fullscreen=true

**Comment by adpe** [comment]
I don't think that this is possible by parameter (without a ugly hack), see Stack Overflow comment https://stackoverflow.com/a/9747340/7262345

**Comment by adpe** [comment]
I don't think that this is possible by parameter (without an ugly hack), see Stack Overflow comment https://stackoverflow.com/a/9747340/7262345

**Comment by McPringle** [comment]
We could at least add documentation of how to accomplish this with startup-parameters of browsers (kiosk mode).

**Comment by adpe** [comment]
You mean something like for Chrome on Windows, create an own shortcut and add this as target `"C:\Program Files\Google\Chrome\Application\chrome.exe" -kiosk --kiosk-printing "http://localhost:8080"`?

**Comment by McPringle** [comment]
Yes. So you know for example how to prepare a Raspberry Pi that you can just attach it to the beamer and start it up, without keyboard and mouse. So you can bring a Pi to a user group meeting or other event and don't need to carry input devices or a notebook with you.
